app/src/main/java/io/gnosis/safe/ui/settings/safe/AdvancedSafeSettingsFragment.kt

```diff
@@ -103,19 +106,41 @@ class AdvancedSafeSettingsFragment : BaseViewBindingFragment<FragmentSettingsSaf
                     modulesContainer.addView(labeledAddress(module.value, module.name ?: getString(R.string.unknown_module), module.logoUri))
                 }
             } ?: run { moduleLabel.visible(false) }
+
+            val contractVersion = SemVer.parse(safeInfo.version)
+            if (contractVersion >= SemVer(1, 1, 3)) {
```

Comment:
the contract version seems to be strange, should it be 1.3.0?

Reply:
yes, it should